cmd/env.go

```diff
@@ -94,18 +94,18 @@ func envRun(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
-	fmt.Printf("export AWS_ACCESS_KEY_ID=%s\n", creds.AccessKeyID)
-	fmt.Printf("export AWS_SECRET_ACCESS_KEY=%s\n", creds.SecretAccessKey)
-	fmt.Printf("export AWS_OKTA_PROFILE=%s\n", profile)
+	fmt.Printf("export AWS_ACCESS_KEY_ID=%q\n", creds.AccessKeyID)
```

Comment:
This will quote with go rules, not bash rules. For instance, `$` is not escaped; if it's `$(cat /etc/passwd)`, that'd get subbed in.

Maybe just use this library: https://github.com/alessio/shellescape

Reply:
I must admit that I don’t see how this can ever be a real issue, as users need to create their own config.

So the values that can be in these export commands are values that are either created locally by the users themselves or come from AWS (after you have authenticated there).

Reply:
@claudenm see the PR that I opened on your branch. The build pipeline uses `govendor` and the dependency that you introduced is not in recorded in the `vendor.json` (which is used by `govendor`).

Reply:
No problem at all!

Reply:
> I must admit that I don’t see how this can ever be a real issue, as users need to create their own config.
> 
> So the values that can be in these export commands are values that are either created locally by the users themselves or come from AWS (after you have authenticated there).

Yep, you're totally right that the risk seems pretty remote. However, considering we're dealing with security-sensitive stuff here, it seems prudent to take the extra step. Thanks for doing it.